packages/nvidia_nat_a365/src/nat/plugins/a365/__init__.py

```
@@ -0,0 +1,30 @@
+from nat.plugins.a365.exceptions import A365ConfigurationError
+from nat.plugins.a365.exceptions import A365Error
+from nat.plugins.a365.exceptions import A365SDKError
+from nat.plugins.a365.exceptions import A365WorkflowExecutionError
+
+__all__ = [
+    "A365Error",
+    "A365AuthenticationError",
+    "A365ConfigurationError",
+    "A365WorkflowExecutionError",
+    "A365SDKError",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)